plugin/storage/integration/integration_test.go

```diff
@@ -118,122 +127,128 @@ func (s *StorageIntegration) IntegrationTestGetOperations(t *testing.T) {
 		t.Log("\t Expected:", expected)
 		t.Log("\t Actual  :", actual)
 	}
-	assert.NoError(t, s.cleanUp())
 }
 
-func (s *StorageIntegration) IntegrationTestGetTrace(t *testing.T) {
-	t.Log("Testing GetTrace ...")
+func (s *StorageIntegration) testGetTrace(t *testing.T) {
```

Comment:
Does it make sense to leave this exposed? 
I feel that it might be useful to have the ability to run a subset of tests when developing new storages

Reply:
we can revisit, but it seems a storage that doesn't support all storage methods is not super useful